### PR TITLE
Digital Assistants vs. Digital Companions

### LINKED ISSUE #813: Digital Assistants vs. Digital Companions
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

In the recent past, the reliance on digital assistants has been enormous, and the number is still expected to grow tremendously over the coming years.
Almost everyone relies on digital assistants such as Siri, Alexa, and Google Assistants in our day to day lives as they make work easier, faster, and simple.
Digital assistants are convenient and easy to use, but the technology that is rapidly evolving and advancing is making us yearn for more convenience, dynamism, and more efficiency in our day to day life experiences.  This leads to the rise of digital companions – the natural evolution of digital assistants.  

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

By the end of this article, the reader should understand;
1.	What is digital assistant?
2.	What is digital companion? 
3.	What are the differences between digital assistants and digital companions? And how are they related?
4.	Applications of the digital assistants and digital companions.
5.	The transition from digital assistants to the digital companions

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by Pius1991**
This fixes #813

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/digital-assistants-vs-digital-companions/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Digital assistants are very easy and convenient to use, but the technology surrounding us is ever rapidly evolving and advancing making digital assistants limited. Due to the limited nature of digital assistants, the need for more seamless, dynamic, and all-around assistants of its kind arises. This has led to the natural evolution of digital assistants known as digital companions.
+
+This article provides an extensive comparison and interrelatedness between digital assistants and digital companions. It enumerates their applications in a real-life situation, their benefits, and future expectation.
+
+### An overview of digital assistants
+
+Digital assistants is an advanced computer program that simulates a conversation with the people who use it, mostly over the internet. Digital assistants use the following to provide a personalized conversation experience;
+
+1. Artificial Intelligence ([IA](https://www.section.io/engineering-education/artificial-intelligence-future/)) – Simulation of human intelligence through programming in a machine to mimic human actions.
```
Comment (lalith1403) [inline articles/digital-assistants-vs-digital-companions/index.md:13]: IA -> AI

File: articles/digital-assistants-vs-digital-companions/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Digital assistants are affordable, convenient, and user-friendly and are primarily used to save time and carry commands. They respond to our requests reactively rather than engaging us proactively.
+
+### An overview of digital companions
+
+Digital companions are just new and improved digital assistants. In a way, they assist us in numerous tasks, and they do so in a personalized, seamless, and fun manner. They are interactive to the user. Digital companions learn and remember what we like or dislike each time we interact, that is individual preferences and needs. They also assist us in understanding and choosing, among many options, the best course of action. In other words, digital companions interact with us proactively as opposed to digital assistants, which do so reactively.
+
+Digital companions have a two-way relationship with their users based on empathy, trust, and anticipation of needs and preferences. They make suggestions that fit our interests and positively influence our behaviors.
```
Comment (lalith1403) [inline articles/digital-assistants-vs-digital-companions/index.md:27]: Have we moved so ahead in technology that digital companions have empathy and trust in them? The anticipation of needs is an actively studied ML problem. Can you clarify on the same? If these are characteristics of an ideal companion, mention that.

File: articles/digital-assistants-vs-digital-companions/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Digital companions are used to automating management and employee experience, thereby making it more accessible. It is also used in performing common tasks such as resetting passwords, system status updates, and outage alerts.
+
+### A comparison of digital assistants and digital companions
+
+- Digital assistants&#39; interactions are primarily command-based, but those of digital companions are both proactive and command-based.
```
Comment (lalith1403) [inline articles/digital-assistants-vs-digital-companions/index.md:33]: Can you create a markdown table for this? Check this [link](https://www.makeuseof.com/tag/create-markdown-table/). A table would make it easier for the reader to compare the two.

File: articles/digital-assistants-vs-digital-companions/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- Digital assistants create short and one-sided exchanges; digital companions create more extended and bidirectional exchanges.
+- Digital assistants use a single primary modality, for instance, voice or text; digital companions use multiple modalities, e.g., voice, movements.
+- Digital assistants are more of generic character and less personality; digital companions are personalized for each user.
+- Digital assistants&#39; experiences are homogenous for all users; digital companions&#39; experiences are personalized for each individual.
+- Digital assistants&#39; actions are executed based on users&#39; commands; digital companions&#39; actions are executed based on context and goals.
+- Digital assistants incorporate very little information about the user while digital companions remember users` needs and preferences.
```
Comment (lalith1403) [inline articles/digital-assistants-vs-digital-companions/index.md:39]: Don't use \` in markdown. Even if you want to use \` in your article, use \\` to represent the character. Moreover, ` is used for code formatting and variables highlighting in markdown. Let me know if you have any doubts regarding this.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article. Congratulations on your first article. Make the changes. We will move it to the next round of review once everything looks good!

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by Pius1991** [comment]
@lalith1403 I have worked on the suggested revisions

**Review by lalith1403 — APPROVED** [review]

**Comment by lalith1403** [comment]
@Pius1991 can you check your avatar image? It says avatar.png.jpg... can you change the extension to avoid any errors?

**Comment by hectorkambow** [comment]
@Pius1991 
Thank you for submitting your 1st article to the EngEd program - we are getting ready to publish your 1st article to the community. 
Please be sure to go over any edits (either I or our peer reviewer made) to get a better sense of the quality we expect moving forward - and see about implementing those into future articles. 🚀 

Be sure to check out our [resources page](https://github.com/section-io/engineering-education/blob/master/new_contributors/resources-page.md) - that has many useful resources to help us all become better technical writers as we continue to explore other great topics. 

cc @lalith1403 
👍 
Thank you

**Pull request merged**